### ISSUE TITLE
Failed successfully

### ISSUE THREAD

**Issue by vinistoisr** [huksley/prometheus-remote-write#6]
with verbose and timing set to true, I get the following response when writing timeseries data:

```
Failed to send write request, error 204 No Content  { timeseries: [ { labels: [Array], samples: [Array] } ] }
Serialized in 1 ms
```

but the data is indeed written successfully to the db. 

index.js seems to treat http status 204 as a failure.  

Changing the segment to include status 204 along with 200 as a successful response might be a fix?

```
if (options?.verbose && r.status != 200 && r.status != 204) {
    logger.warn("Failed to send write request, error", r.status + " " + r.statusText + " " + text, writeRequest);
} else if (options?.verbose) {
    if (options?.timing) {
        logger.info(
            "Write request sent",
            r.status + " " + r.statusText + " in",
            Date.now() - start2,
            "ms",
            writeRequest
        );
    } else {
        logger.info("Write request sent", r.status + " " + r.statusText + " " + text, writeRequest);
    }
}
```
https://github.com/huksley/prometheus-remote-write/blob/63a391dfcd6daf12e013acf3bf7d6288b79440af/index.js#L111

**Comment by huksley** [comment]
hmm, which provider you use?

**Comment by vinistoisr** [comment]
I'm writing to VictoriaMetrics vm_agent according to these docs https://docs.victoriametrics.com/vmagent/#prometheus-remote_write-proxy

**Comment by huksley** [comment]
Published npmjs package 0.4.1, could you test?

**Comment by vinistoisr** [comment]
thank you, that solved the issue.
